### PR TITLE
tbi-deps-next

### PR DESCRIPTION
**Pull request by bcdarwin**
- flake.nix: add additional supportedNvidiaDrivers
 - flake.lock: bump nixpkgs
 - emacsWithPackages: remove
 - add ITK (bloats the container by ~0.8G uncompressed though)
 - add finetuning-scheduler for pytorch-lightning training experiments
 - add ibis framework and some backend deps (polars, postgres, duckdb)
 - restore ydata-profiling (was temporarily broken)
 - add great-tables (for notebook output formatting)
 - ignite: remove (since we only use pytorch-lightning)

### CODE REVIEW

**Comment by bcdarwin** [comment]
@cfhammill ready to go
